### ISSUE TITLE
Multi sites version

### ISSUE THREAD

**Issue by pulsarinformatique** [Kicktemp/kickgdpr#7]
Hi

I believe it would be useful to extend the legal notices filtering not only on the language but also on the URL for multi sites versions.

Actually we have several projects (with seblod) with several sites on the same joomla instance. The only way to diffirentiate them is on the URL (different domains).

For these projects each site has a different legal notice but currently the plugin is unable to select a separate content  / menu item depending on the url

thanks for sharing

cyril

**Comment by tecpromotion** [comment]
Hi @pulsarinformatique 

I have no idea how to trigger this because we do not use Seblod, but we like to look at it.

**Comment by pulsarinformatique** [comment]
It would have nothing to so with seblod here, just with the domain.
I gave this info to tell how the seblod multi sites system works
thanks

**Comment by tecpromotion** [comment]
Joomla! does not support multisites itself, so it will probably be difficult to realize this independently of a third party extension. But we'll look at that and give you feedback.

**Comment by pulsarinformatique** [comment]
thanks

**Comment by tecpromotion** [comment]
Unfortunately, we can not rebuild that way. Seblod is just an extension which brings multisite features. As long as the Joomla! Core does not support multisite, it will be difficult to consider all points here.

**Comment by pulsarinformatique** [comment]
Yes I understand but let's forget Seblod here please: Can't we just filter on the URL as many other extensions do (advanced module manager for example)

thanks

cyril

**Comment by tecpromotion** [comment]
At the present time, we are directing the development in a different direction. But we will try to incorporate this into the further development.
